### ISSUE TITLE
release: v8.4.0-DMR

### ISSUE THREAD

**Issue by yinsustart** [pingcap/tidb#57048]
**Disclaimer**
The dates published in this issue are for planning purposes only. It is intended solely to help you plan your projects. A date that exists in the future should not be considered a firm release date as the release and timing of releases are subject to change at any time and are at PingCAP's sole discretion.

Release process checklist
Candidate SHA:
TiDB: 1a9f0fa25580907d3266dc3edf000ba24379a600
TiKV: 22bbfeb958936ab75dda1f60ca6bdc1cc0bb5231
PD: fc6e45cc348da147e90fbac6cc665366b11db8cd
TiFlash: cea728d5d2bad7766dfc5e97cdf8ef16fe4a791f
tidb-tools: d226440121147098eb5eb99cbc1efb94092ec68e
tiflow: ad58c8b8acd2b09b73dd3c3cac595c31fb35ccfd
ng-monitoring: 0c5bbdc74ec73d48e0c0ab907d6815805292fcf9
enterprise-plugin: 580afe5fe65842bf8f69d78d35034b715ba50e28
dumpling: 1a9f0fa25580907d3266dc3edf000ba24379a600
tidb-dashboard: 776012892d7b0e7c2b06d66dd40b5ed8861439e3
monitoring: 0e0e38ed3a6d8c1e2e159b8e8cbb8793e6ac6781

Working Tags: v8.4.0

Check the diff in the current version:

- [x] 0. Announce the release plan

- [x] 1. Create release-8.4 branch: Oct. 10, 2024

- [x] 2. Code freeze: Oct. 10, 2024

- [x] 3. Qualifying the release.

- [x] 4. Docs preparing for release.

- [x] 5. The release qualified.

- [x] 6. Docs ready for release.

**release date: Nov. 2024**

Do not proceed below until the release date.

- [x] 7. Publish TiDB Release

> fill in Candidate SHA above

> fill in Make and Publish Build above

- [x] 8. Update version numbers

- [x] 9. Update docs (handled by Docs team)

- [x] 10. Update the official website

**Comment by yinsustart** [comment]
We are sorry to announce that TiDB v8.4.0 release will be postponed from October to November.

Reason and next plan:
Reason

- We've discovered [a TiKV OOM issue](https://github.com/tikv/tikv/issues/17696) related to CDC, Titan, and PiTR during regression testing.While the root cause is still being investigated, we have identified the direct cause as CDC tasks accumulating in the "apply" to "cdc" channel due to slow processing of `old_value_cb`, which is overloading the CDC thread.

Next plan

- Will use a bisecting approach to identify suspicious commits. Concurrently, the CDC team will take steps to alleviate the OOM issue.

- We are strictly following the release schedule, but the current issue has the potential to introduce OOM issues that carry quality risks. After a careful assessment, it needs more time to resolve it before release.

**Comment by wuhuizuo** [comment]
## Candidate SHAs

- TiDB: 1a9f0fa25580907d3266dc3edf000ba24379a600
- TiKV: 22bbfeb958936ab75dda1f60ca6bdc1cc0bb5231
- PD: fc6e45cc348da147e90fbac6cc665366b11db8cd
- TiFlash: cea728d5d2bad7766dfc5e97cdf8ef16fe4a791f
- binlog: // deprecated.
- tidb-tools: d226440121147098eb5eb99cbc1efb94092ec68e
- tiflow: ad58c8b8acd2b09b73dd3c3cac595c31fb35ccfd
- ng-monitoring: 0c5bbdc74ec73d48e0c0ab907d6815805292fcf9
- enterprise-plugin: 580afe5fe65842bf8f69d78d35034b715ba50e28
- dumpling: 1a9f0fa25580907d3266dc3edf000ba24379a600
- tidb-dashboard: 776012892d7b0e7c2b06d66dd40b5ed8861439e3
- monitoring: 0e0e38ed3a6d8c1e2e159b8e8cbb8793e6ac6781

## Working Tags

v8.4.0
